test/senderSpec.js

```diff
@@ -28,6 +30,51 @@ describe('Sender Module', function () {
     timekeeper.reset()
   })
 
+  describe('createRequest', function () {
+    beforeEach(function () {
+      this.hmacHelper = createHmacHelper()
+      this.sender = createSender({
+        client: this.client,
+        uuidSeed: Buffer.from('f73e2739c0f0ff4c9b7cac6678c89a59ee6cb8911b39d39afbf2fef9e77bc9c3', 'hex')
+      })
+      this.kep = {
+        destination_account: 'ilpdemo.blue.bob.~kep.ZfiUdFj-tVw.HHxfobwe-sscE5rKUrCksA',
+        shared_secret: '8qAZtXsrK8Lz_BTSv4D2zA'
+      }
+    })
+
+    it('should generate a payment request', function () {
+      const request = this.sender.createRequest(Object.assign({}, this.kep, {
+        destination_amount: '1'
+      }))
+
+      assert.match(request.address, new RegExp('^' + this.kep.destination_account))
+      assert.equal(request.amount, '1')
+      assert.equal(request.expires_at, '1970-01-01T00:00:30.000Z')
+    })
+
+    it('should encrypt the payment request data and store as a base64-encoded blob', function () {
```

Comment:
I'm not sure this makes sense. If you encrypt data that you're giving to the sender, just for them to send it back to you, what's the point of putting it there at all?